pkg/gitprovider/bitbucketserver.go

```diff
@@ -246,7 +246,7 @@ func (g *BitbucketServerGitProvider) GetUser() (*GitUser, error) {
 
 	username := res.Header.Get("X-Ausername")
 	if username == "" {
-		return nil, fmt.Errorf("X-Ausername header is missing")
+		return nil, fmt.Errorf("header X-Ausername is missing")
```

Comment:
Revert this, it is okay that error start with a capital letter if it is a header key.